types/underscore/underscore-tests.ts

```diff
@@ -685,6 +686,7 @@ function chain_tests() {
     .value();
   var paired: Array<[string, number]> = _.chain({a: 1, b: 2}).pairs().value();
   var inverted: {[key: string]: string} = _.chain({a: 'b', b: 'a'}).invert().value();
+  var invertedArray: {[key: string]: number} = _.chain(['a', 'b', 'c']).invert().value();
```

Comment:
This is probably the reason why you made `_.invert` `_Chain<any, Dictionary<any>>`... my bad.